RadixWallet/Features/AccountHistory/TransactionHistory+View.swift

```diff
@@ -0,0 +1,611 @@
+									ProgressView()
+										.padding(.small1)
+								}
+
+								ForEach(viewStore.sections) { section in
+									SectionView(section: section)
+										.onAppear {
+											store.send(.view(.sectionAppeared(section.id)))
+										}
+										.onDisappear {
+											guard !viewStore.didDismiss else { return }
+											store.send(.view(.sectionDisappeared(section.id)))
+										}
+								}
+
+								Rectangle()
+									.fill(.clear)
+									.frame(height: .medium3)
+									.overlay {
+										if viewStore.loading.isLoading, viewStore.loading.parameters.backwards {
+											ProgressView()
+										}
+									}
+							}
+							.padding(.bottom, .medium3)
+						}
+						.scrollIndicators(.never)
+						.coordinateSpace(name: View.coordSpace)
+					}
+					.background {
+						if viewStore.showEmptyState {
+							Text(L10n.TransactionHistory.noTransactions)
+								.textStyle(.sectionHeader)
+								.foregroundStyle(.app.gray2)
+						}
+					}
+					.background(.app.gray5)
+					.overlayPreferenceValue(PositionsPreferenceKey.self, alignment: .top) { positions in
+						let rect = positions[View.accountDummy]
+						ZStack(alignment: .top) {
+							if let rect {
+								accountHeader
+									.offset(y: rect.minY)
+							}
+
+							let scrollBarOffset = max(rect?.maxY ?? 0, 0)
+							VStack(spacing: .small2) {
+								HScrollBar(items: viewStore.availableMonths, selection: selection)
+
+								if let filters = viewStore.activeFilters.nilIfEmpty {
+									ActiveFiltersView(filters: filters) { id in
+										store.send(.view(.filterCrossTapped(id)), animation: .default)
+									}
+								}
+							}
+							.padding(.top, .small2)
+							.padding(.bottom, .small1)
+							.background(.app.white)
+							.offset(y: scrollBarOffset)
+						}
+					}
+					.onReadPosition(View.accountDummy) { rect in
+						if rect.minY > 50 {
+							store.send(.view(.pulledDown))
+						}
+					}
+					.clipShape(Rectangle())
+					.toolbar {
+						ToolbarItem(placement: .topBarLeading) {
+							CloseButton {
+								store.send(.view(.closeTapped))
+							}
+						}
+						ToolbarItem(placement: .topBarTrailing) {
+							Button(asset: AssetResource.transactionHistoryFilterList) {
+								store.send(.view(.filtersTapped))
+							}
+						}
+					}
+				}
+				.navigationTitle(L10n.TransactionHistory.title)
+				.navigationBarTitleDisplayMode(.inline)
+			}
+			.onAppear {
+				store.send(.view(.onAppear))
+			}
+			.destinations(with: store)
+			.ignoresSafeArea(edges: .bottom)
+		}
+
+		private static let coordSpace = "TransactionHistory"
+		private static let accountDummy = "SmallAccountCardDummy"
+	}
+
+	struct SectionView: SwiftUI.View {
+		let section: TransactionHistory.State.TransactionSection
+
+		var body: some SwiftUI.View {
+			Section {
+				ForEach(section.transactions, id: \.self) { transaction in
+					TransactionView(transaction: transaction)
```

Comment:
There is no onTap handler? Shouldn't we navigate to the dashboard when user tapped on a transaction id?
@jakub-rdx is it handled on Android?

Reply:
on Android we open dashboard for this tx details in browser